c/meterpreter/source/extensions/stdapi/server/sys/process/process.c

```diff
@@ -860,6 +860,91 @@ DWORD request_sys_process_kill(Remote *remote, Packet *packet)
 	return ERROR_SUCCESS;
 }
 
+DWORD request_sys_process_nkill(Remote *remote, Packet *packet)
+{
+	Packet *response = packet_create_response(packet);
+	DWORD result = ERROR_SUCCESS;
+	if (!response)
+	{
+		return ERROR_SUCCESS;
+	}
+
+	PCHAR NameKl = packet_get_tlv_value_string(packet, TLV_TYPE_PROCESS_NAME);

```

Comment:
As per the existing functionality, would it not be better to support a list of process names rather than just one?